edb/lang/edgeql/compiler/setgen.py

```diff
@@ -630,10 +633,6 @@ def computable_ptr_set(
         ctx: context.ContextLevel) -> irast.Set:
     """Return ir.Set for a pointer defined as a computable."""
     ptrcls = rptr.ptrcls
-
-    # Must use an entirely separate context, as the computable
-    # expression is totally independent from the surrounding query.
-    subctx = stmtctx.init_context(schema=ctx.schema)
     self_ = rptr.source
```

Comment:
Maybe we can use a bit less confusing name than `self_`?

Reply:
Renamed to `source_set`